docs/4.operation/6.mastermind.md

```diff
@@ -0,0 +1,13 @@
+## FIWARE Mastermind
+
+MasterMind is an ongoing development effort to provide a solution to
+easily configure and orchestrate FIWARE NGSI-based services. The service
+is provided to FIWARE Lab end-users as a centralised service. No
+specific action is needed by single FIWARE Lab Node Administrators.
+Currently the service in under development, so it is not yet deployed in
```

Comment:
Check this sentence, probably it is obsolete...

"Currently the service in under development..."